packages/graphile-build-pg/src/plugins/PgColumnDeprecationPlugin.js

```diff
@@ -4,28 +4,19 @@ import type { Plugin } from "graphile-build";
 export default (function PgColumnDeprecationPlugin(builder) {
   builder.hook("GraphQLObjectType:fields:field", (field, build, context) => {
     const {
-      scope: {
-        isPgRowType,
-        isPgCompoundType,
-        pgIntrospection: table,
-        pgFieldIntrospection,
-      },
+      scope: { pgFieldIntrospection },
     } = context;
     if (
-      !(isPgRowType || isPgCompoundType) ||
-      !table ||
-      table.kind !== "class" ||
       !pgFieldIntrospection ||
-      !pgFieldIntrospection.tags
+      !pgFieldIntrospection.tags ||
+      !pgFieldIntrospection.tags.deprecated
```

Comment:
This is a very broad check...

- Applies to CRUD mutations if the table is deprecated... fine ✅ 
- Applies to custom mutations if the function is deprecated... fine ✅ 
- Applies to columns on tables... fine ✅ 
- Applies to computed columns... fine ✅ 
- Applies to custom queries... fine ✅ 
- Applies to views... fine ✅ 

Okay, looks good. I can't think of a good reason this would not be desired.